internal/tools/uvmboot/lcow.go

```diff
@@ -159,7 +168,21 @@ func init() {
 }
 
 func createLCOWOptions(ctx context.Context, c *cli.Context, id string) (*uvm.OptionsLCOW, error) {
-	options := uvm.NewDefaultOptionsLCOW(id, "")
+	opt, err := oci.SpecToUVMCreateOpts(ctx,
```

Comment:
Lot of the existing arguments (`bootFilesPathArgName`, `kernelDirectArgName` etc.) to the lcow command overlap with annotations. If we are supporting annotations, should we get rid of these duplicate arguments? Or maybe deprecate them for now?